DetectorDescription/Parser/test/run_testDoc.sh

```diff
@@ -3,11 +3,11 @@ test=testtestDoc
 function die { echo Failure $1: status $2 ; exit $2 ; }
 pushd ${LOCAL_TMP_DIR}
   export mecpath=${PATH}
-  export PATH=${LOCAL_TOP_DIR}/test/${SCRAM_ARCH}/:${PATH}
+  test -z "${CMSSW_RELEASE_BASE}" && echo CMSSW_RELEASE_BASE is not set.
+  export PATH=${CMSSW_RELEASE_BASE}/test/${SCRAM_ARCH}/:${LOCAL_TOP_DIR}/test/${SCRAM_ARCH}/:${PATH}
   cd ${LOCAL_TEST_DIR}
-#  cp ${LOCAL_TOP_DIR}/test/${SCRAM_ARCH}/testDoc .
-  echo ${test}testDoc ------------------------------------------------------------
-  testDoc testConfiguration.xml|| die "testDoc" $?
+  echo ${test} testDoc ------------------------------------------------------------
+  testDoc testConfiguration.xml || die "testDoc" $?
```

Comment:
@cvuosalo , the `testDoc` is not part of this Package. So calling it here is not going to work. I would suggest to move this unit tst in to `DetectorDescription/RegressionTest` package (which has `testDoc`). If you do not want to move this then you need to do something like [a]. Note that one has to lookin local directory (i.e. CMSSW_BASE) , release directory (i.e. CMSSW_RELEASE_BASE) or full release path e.g. in case of patch release (CMSSW_FULL_RELEASE_BASE). I would still suggest to move this test to `DetectorDescription/RegressionTest` . In that case `scram n runtests` will find it.

```
testDoc=testDoc
for d in $CMSSW_BASE $CMSSW_RELEASE_BASE $CMSSW_FULL_RELEASE_BASE ; do
  if [ -x $d/test/${SCRAM_ARCH}/testDoc ] ; then 
    testDoc=$d/test/${SCRAM_ARCH}/testDoc
    break
  fi
done
$testDoc testConfiguration.xml
```

Reply:
@smuzaffar Thanks for the code snippet. That solves the problem. I don't want to move the test. This whole package may go away when the old DD is deleted next year. Also, this test is for this package, so it does belong here, not in `DetectorDescription/RegressionTest`.